build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'java'
```

Comment:
Prefer applying plugins using the `plugins {}` block, see https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block
```suggestion
plugins {
    id 'java'
}
```